en/docs/reference/connectors/rabbitmq-amqp-1.0-connector/rabbitmq-amqp-1.0-connector-example.md

```
@@ -0,0 +1,283 @@
+
+2. Now we need to configure the necessary parameters of the `publishMessage` operation. For some of the fields, we will use Synapse expressions to map values from the sample request defined in [Create Sample Request Payload](#create-sample-request-payload).
+    - Destination Queue/Exchange Section
+        - **Queue**: `${payload.queue}`
+        - **Queue Type**: QUORUM
+        - **Queue Auto Declare** : true
+        
+    - Request Message Settings
+        - **Content Type**: TEXT
+        - **Request Body**: `${payload.message}`
+
+   - Publisher Client Settings
```

markdownlint-cli2 (0.20.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 4; Actual: 3

(MD005, list-indent)